### ISSUE TITLE
Spereate commissions for equtites / futures?

### ISSUE THREAD

**Issue by npezolano** [quantopian/zipline#604]
Is it possible to set separate commissions for different asset classes?

**Comment by warren-oneill** [comment]
As far as I'm aware no. The commission is set for an algorithm before a run with the [set_commission](https://github.com/quantopian/zipline/blob/master/zipline/algorithm.py#L910) function and is fixed for the entire run. It would make a great addition though!

**Comment by npezolano** [comment]
@warren-oneill Is the idea that you can only backtest one asset class at a time or more than one? If both you would need this feature.

**Comment by jfkirk** [comment]
This is a forthcoming feature. We are building toward allowing a single algorithm to trade multiple Asset classes, but you are absolutely right that this is a constraint in the current version. For now, you would have to either:

1. Use a commission rate somwhere between Equities and Futures (which is unrealistic)
2. Only trade on asset class per algo

**Comment by jfkirk** [comment]
Self-assigning.

**Comment by jgoode21** [comment]
Yeah I give a plus one for this, as I've written Zipline algorithms that trade equities and their options. The options have different commissions that their underlying equities. I just deducted the difference from my ending PnL, but it wasn't the cleanest work around.

**Comment by jfkirk** [comment]
@jgoode21 Makes sense - you shouldn't have to do such a workaround, so we'll work toward getting you this feature.
